src/config/mascotMessages.ts

```diff
@@ -0,0 +1,42 @@
+export type Message = {
+  content: string,
+  faceExpression?: any, //TODO specify
```

Comment:
Please implement some example FaceExpressions instead of using `any`.
Example:
`export type FaceExpression = 'happy' | 'sad' | 'neutral';`